FEXCore/Source/Interface/Core/JIT/Arm64/VectorOps.cpp

```diff
@@ -5009,42 +5009,50 @@ DEF_OP(VTBX1) {
   if (Dst != VectorSrcDst) {
     switch (OpSize) {
       case 8: {
-        mov(Dst.D(), VectorSrcDst.D());
+        mov(VTMP1.D(), VectorSrcDst.D());
+        tbx(VectorSrcDst.D(), VectorTable.Q(), VectorIndices.D());
```

Comment:
This tbx and the following two which are using the VTMP1 are incorrectly still using `VectorSrcDst` as their destination. Should use VTMP1 to be correct here.

Reply:
Yes, we want to improve our RA to support constraints to handle these cases. It's on the roadmap.